utilities/docker/geppetto-persistence/Dockerfile

```diff
@@ -5,6 +5,12 @@ ARG aKey
 ENV aKey=${aKey}
 ARG sKey
 ENV sKey=${sKey}
+ARG persistenceBranch
```

Comment:
@tarelli At time of comment, it would have failed without those parameters. Last commit makes it so development branch is used if those parameters are not passed.